pkg/ccloudv2/utils.go

```diff
@@ -24,7 +24,7 @@ const (
 )
 
 var (
-	ByocSupportClouds = []string{"aws", "azure"}
+	ByocSupportClouds = []string{"AWS", "AZURE"}
```

Comment:
Oops, I see how this is getting used now. This is a suggestion for what the user should type, so it should be lowercase. Sorry for the confusion!

```suggestion
	ByocSupportClouds = []string{"aws", "azure"}
```